cpp/src/parquet/parquet_constants.h

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Put those files in a `thrift_generated` subdirectory perhaps?

Reply:
Would that fix the build issue, or you mean just to be explicit that they're generated?

Reply:
Just be explicit that they're generated.

Reply:
Since we already have src/generated that seems like an OK place